src/app/component/layout/components/header/header.component.scss

```diff
@@ -487,11 +481,24 @@ $screen-320: " screen and (max-width : 320px)";
   }
 }
 
+@media screen and (max-width: 767px) {
+  .main {
+    .main-container {
+      margin: 0 1.95%;
```

Comment:
why is 1.95 %?

Reply:
> why is 1.95 %?

According to the mockup, margin is 15 px at the screen width of 768px. 15*100/768 is close to 1.95.